api/WebTransportBidirectionalStream.json

```
@@ -65,6 +65,40 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "returns_WebTransportReceiveStream": {
+          "__compat": {
+            "description": "Returns <code>WebTransportReceiveStream</code> (earlier spec returned <code>WritableStream</code>)",
+            "spec_url": "https://w3c.github.io/webtransport/#dom-webtransportbidirectionalstream-readable",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "121"
```

114 for this. 121 was for part of the type.
```suggestion
                "version_added": "114"
```